src/lib/dai.c

```diff
@@ -129,55 +129,99 @@ void dai_group_put(struct dai_group *group)
 }
 
 #if CONFIG_ZEPHYR_NATIVE_DRIVERS
+const struct device *dai_get_zephyr_driver(uint32_t type, uint32_t index)
+{
+	const struct device *drv = NULL;
```

Comment:
is this common to call a `struct device` pointer `drv`?

Reply:
yeah I think in zephyr the device = driver

Reply:
strange, I haven't found many such examples in Zephyr. I found indeed a couple of them, e.g. in drivers/serial/uart_lpc11u6x.c but that's about it, most use `struct device *dev` or similar. Device and driver should be pretty different things.

Reply:
I think we are getting into philosophical turf here... I mean "device" is the type of the struct that you instantiate to use the code associated with it. Most of the definitions are under "drivers" directory in zephyr... So maybe driver is "device + code" conceptually??? personally I think device = driver as you cant do anything without the other concept. Anyway I can change it to dev, really doesnt matter to me.